utils/python/CIME/XML/component.py

```diff
@@ -6,6 +6,8 @@
 from entry_id import EntryID
 from CIME.utils import expect, get_cime_root, get_model
 
+logger = logging.getLogger(__name__)
```

Comment:
yes they do, but by default all loggers go to the same handler.  try running with the different levels to see how each is treated.